### ISSUE TITLE
quest book pages vanish after crash

### ISSUE THREAD

**Issue by foxdarts** [davqvist/Exoria#196]
![quest book died](https://user-images.githubusercontent.com/39364370/40246561-36e6aea8-5a97-11e8-90ae-6a11365a5b1f.jpg)
got a crash when attempting to shift click out of the chisel and when I loaded back into my game the pages within the quest book were gone

**Comment by davqvist** [comment]
Try to click "Done" and see if it opens anything.
If not, try the command "/bq_admin default load"

**Comment by foxdarts** [comment]
I tried both and it says I don't have permission to do that command

**Comment by davqvist** [comment]
Yes of course, you have to "Open to LAN" and enable cheats to do that

**Comment by foxdarts** [comment]
for whatever reason I didnt think to try it on a lan . that worked but it wiped all my quest progress.

**Comment by davqvist** [comment]
What? That sounds weird. Have never heard of that. Something strange is going on

**Comment by foxdarts** [comment]
eh its ok. I still have everything that I used to complete them so it is all good. idk why it wiped though
